app/src/main/java/eu/kanade/tachiyomi/ui/library/DisplayBottomSheet.kt

```diff
@@ -72,6 +73,15 @@ class DisplayBottomSheet(private val controller: LibraryController) : BottomShee
                     settings_scroll_view.paddingTop + settings_scroll_view.paddingBottom
             close_button.visibleIf(isScrollable)
         }
+        updateRadioGroupScrollBar()
+    }
+
+    private fun updateRadioGroupScrollBar() {
+        show_hide_auto_scrollbar.bindToPreference(preferences.showHideScrollBar()) {
+            controller.updateScrollBar()
+            if (sheetBehavior.state == BottomSheetBehavior.STATE_COLLAPSED)
+                dismiss()
+        }
```

Comment:
this logic should be in the initGeneralPreferences method like the other preferences